change/@ni-nimble-components-225009b9-77d6-4cbd-a94e-d9ba2751b047.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note that our build pipeline enforces some basic formatting rules on markdown and it's currently failing. We have [some instructions](https://github.com/ni/nimble/blob/cc709bb825c03fa299e401b152df19ae3c243e1a/CONTRIBUTING.md#linting) about how to configure VSCode to automatically fix these for you. They're written for a developer audience though so feel free to reach out if you need help setting it up.

Reply:
silvenon.mdx is deprecated so I can't install that, but I have everything else installed. I ran npm run format and npm run lint after I committed my most recent change. But now when I try to "Sync changes" it says "Can't push refs to remote. Try running "Pull" first to integrate your changes."